lib/src/Components/Map/Subcomponents/Controls/SearchControl.tsx

```diff
@@ -113,6 +113,7 @@ export const SearchControl = () => {
                 autoComplete='off'
                 value={value}
                 className='tw:input tw:input-bordered tw:h-12 tw:grow tw:shadow-xl tw:rounded-box tw:pr-12 tw:w-full'
+                data-cy='search-input'
```

Comment:
As already suggested we should remove those tags when building the library normally. Since this is work and the PR is big enough I suggest to make an Issue for it, so it cannot be forgotten.

Reply:
Will be taken care of in #418